commercetools-convenience/src/main/java/io/sphere/sdk/sequencegenerators/CustomObjectBigIntegerNumberGeneratorConfig.java

```diff
@@ -1,25 +1,48 @@
 package io.sphere.sdk.sequencegenerators;
 
 import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.customobjects.CustomObject;
 import io.sphere.sdk.models.Base;
 
 import java.math.BigInteger;
 
 /**
- * Represents the configuration for the Sequential number generator .
+ * Configuration for the Sequential number generator.
  *
- * @see CustomObjectBigIntegerNumberGeneratorConfig
+ * @see CustomObjectBigIntegerNumberGeneratorConfigBuilder
  */
-
 public final class CustomObjectBigIntegerNumberGeneratorConfig extends Base {
 
     private final SphereClient sphereClient;
+
+    /**
+     *
+     * If there is a concurrency exception, the generator retries to generate the number. This parameter sets the maximum number of retries.
+     * (by default is is set to 100 in the config builder)
+     *
+     * @see CustomObjectBigIntegerNumberGeneratorConfigBuilder
+     */
     private final int maxRetryAttempts;
```

Comment:
I added the javadoc for the parameters in the config builder instead of the config class, is it correct solution?